client/components/social-buttons/google.js

```diff
@@ -123,8 +125,19 @@ class GoogleLoginButton extends Component {
 				if ( 'idpiframe_initialization_failed' === error.error ) {
 					// This error is caused by 3rd party cookies being blocked.
 					this.setState( {
-						error: translate(
-							'Please enable "third-party cookies" to connect your Google account.'
+						error: createInterpolateElement(
+							translate(
+								'Please enable "third-party cookies" to connect your Google account. To find out how to do this <learn_more_link>learn more here</learn_more_link>.'
```

Comment:
```suggestion
								'Please enable "third-party cookies" to connect your Google account. To learn how to do this, <learn_more_link>click here</learn_more_link>.'
```

i think this might be more clear